features/src/main/scala/com/salesforce/op/utils/spark/OpVectorMetadata.scala

```diff
@@ -50,7 +50,7 @@ class OpVectorMetadata private
 (
   val name: String,
   col: Array[OpVectorColumnMetadata],
-  val history: Map[String, FeatureHistory]
+  val history: Map[String, FeatureHistory] // TODO fix map -> causes problems when multiple vectorizers used on feature
```

Comment:
I thought this was still an issue? No?

Reply:
it is an issue - that is why I put the todo